src/IssueDb/Crawling/CrawledIndex.cs

```diff
@@ -190,6 +190,7 @@ public async Task SaveAsync(string path)
                         {
                             writer.Write(stringIndexer(kv.Key));
                             writer.Write(stringIndexer(kv.Value.Lead));
+                            writer.Write(stringIndexer(kv.Value.Pod));
```

Comment:
You should bump the format version in line 15. This prevents the index from loading an old format.